plugins/woocommerce/src/Admin/API/Reports/Orders/Stats/DataStore.php

```diff
@@ -517,6 +525,8 @@ public static function update( $order ) {
 				'order_id'           => $order->get_id(),
 				'parent_id'          => $order->get_parent_id(),
 				'date_created'       => $order->get_date_created()->date( 'Y-m-d H:i:s' ),
+				'date_paid'          => $order->get_date_paid() ? $order->get_date_paid()->date( 'Y-m-d H:i:s' ) : null,
```

Comment:
Do we anticipate needing this date in GMT as well?  (Not trying to lead to an answer, I'm really not sure).

Reply:
I'm not sure as well. Maybe it's ok to add it later if needed?